go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/fluxcd/helm-controller/api v1.4.4
 	github.com/giantswarm/apiextensions-application v0.6.2
 	github.com/giantswarm/cluster-standup-teardown/v2 v2.2.2
-	github.com/giantswarm/clustertest/v2 v2.1.2
+	github.com/giantswarm/clustertest/v2 v2.1.3-0.20251103123957-ea28b1cca132
```

Comment:
Update to a released version plus also bump `clustertest` in `cluster-standup-teardown`, cut a release and bump it in this place.